terms_dynamic.go

```diff
@@ -30,6 +30,9 @@ import (
 )
 
 func loadDynamicTerminfo(term string) (*terminfo.Terminfo, error) {
+	if term == "" {
+		return nil, ErrTermNotFound
```

Comment:
nit: maybe wrap this with "term not set" or something like that?

```go
return nil, fmt.Errorf("%w: term not set",ErrTermNotFound)
```

that could make it even easier to debug.

Reply:
I've made the requested change to the pull request.